bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/layout/TabFolderLayout.java

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2005 IBM Corporation and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.swt.layout;
+
+import org.eclipse.swt.*;
+import org.eclipse.swt.graphics.*;
+import org.eclipse.swt.widgets.*;
+
+/**
+ * This layout controls the position and size
+ * of the children of a tab folder.
+ *
+ * @since 2.1
```

Comment:
Is this value correct?

Reply:
I havent updated the content of the file actually. What should be the value ideally?

Reply:
It should be the current version of the bundle. If the bundle does currently have a 0 micro version the minor version should be incremented because the is new API and that new version should be used. I’m on an iPhone so it’s hard to check.